core/kazoo_apps/src/kapps_controller.erl

```diff
@@ -36,9 +36,14 @@ start_link() ->
 
 -spec ready() -> boolean().
 ready() ->
-    Configured = start_which_kapps(),
-    Running = [kz_term:to_binary(App) || App <- running_apps()],
-    lists:subtract(Configured, Running) =:= [].
+    Configured = [kz_term:to_binary(App) || App <- start_which_kapps()],
```

Comment:
@jamesaimonetti can we redo the logic here? log lines remove or move the log in line 187 ? there's a lot of messaging in the log, not needed for now
```
03:47:52.963 info kapps_controller.187 starting applications from default configuration
03:47:53.064 info kapps_controller.187 starting applications from default configuration
03:47:53.165 info kapps_controller.187 starting applications from default configuration
03:47:53.266 info kapps_controller.187 starting applications from default configuration
03:47:53.367 info kapps_controller.187 starting applications from default configuration
03:47:53.468 info kapps_controller.187 starting applications from default configuration
03:47:53.569 info kapps_controller.187 starting applications from default configuration
03:47:53.670 info kapps_controller.187 starting applications from default configuration
03:47:53.771 info kapps_controller.187 starting applications from default configuration
03:47:53.872 info kapps_controller.187 starting applications from default configuration
03:47:53.973 info kapps_controller.187 starting applications from default configuration
03:47:54.074 info kapps_controller.187 starting applications from default configuration
03:47:54.175 info kapps_controller.187 starting applications from default configuration
03:47:54.276 info kapps_controller.187 starting applications from default configuration
03:47:54.377 info kapps_controller.187 starting applications from default configuration
```